examples/generate_totp/generate_totp.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	wallet "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples/exampleutil"
+	"github.com/bitcoin-sv/spv-wallet/models"
+)
+
+func main() {
+	const aliceXPriv = "xprv9s21ZrQH143K4JFXqGhBzdrthyNFNuHPaMUwvuo8xvpHwWXprNK7T4JPj1w53S1gojQncyj8JhSh8qouYPZpbocsq934cH5G1t1DRBfgbod"
```

Comment:
For now, this constant is unused.

```suggestion
	const aliceXPriv = examples.UserXPriv
```

and then use it for:
```golang
usersAPI, err := wallet.NewUserAPIWithXPriv(exampleutil.NewDefaultConfig(), aliceXPriv)
```

This should point out who is party A (Alice) and B (Bob)